coderd/tailnet.go

```diff
@@ -97,6 +98,18 @@ func NewServerTailnet(
 		agentConnectionTimes: map[uuid.UUID]time.Time{},
 		agentTickets:         map[uuid.UUID]map[uuid.UUID]struct{}{},
 		transport:            tailnetTransport.Clone(),
+		connsPerAgent: prometheus.NewGauge(prometheus.GaugeOpts{
+			Namespace: "coder",
+			Subsystem: "servertailnet",
+			Name:      "open_tcp_connections",
```

Comment:
I realize we only use servertailnet for HTTP proxying at present, but what do you think about changing this to `open_connections` and making the network a label (e.g. `network=tcp`?  Like, I dunno, what if some customer wants us to proxy QUIC or some shit?

Reply:
Yeah that's a good point. If we were to keep it like this we would definitely preclude ourselves from expanding this without a breaking change. Adding a `network=tcp` with it really only ever being tcp for the forseeable future basically has the same implications and usability as not having it, probably a good change.